spec/lib/solana_rpc_ruby/api_client_spec.rb

```diff
@@ -88,6 +88,18 @@
         end
       end
 
+      describe 'Net::HTTPTooManyRequests' do
+        it 'raise error correctly' do
```

Comment:
Did you run rspec? I am getting an error:
```  
1) SolanaRpcRuby::ApiClient#call_api error handling Net::HTTPTooManyRequests raise error correctly
     Failure/Error:
       receive(:request).with(an_instance_of(Net::HTTP::Post))
         .and_raise(Net::HTTPTooManyRequests.new('Net::HTTPTooManyRequests', 'Response'))

 ArgumentError:
       wrong number of arguments (given 2, expected 3)
     # ./spec/lib/solana_rpc_ruby/api_client_spec.rb:95:in `new'
     # ./spec/lib/solana_rpc_ruby/api_client_spec.rb:95:in `block (5 levels) in <top (required)>'
```

Reply:
Oops, should be fixed now.

Reply:
I'm getting new error 
```
1) SolanaRpcRuby::ApiClient#call_api error handling Net::HTTPTooManyRequests raise error correctly
     Failure/Error:
       expect do
         described_class.new.call_api(body: {}, http_method: :post)
       end.to raise_error(SolanaRpcRuby::ApiError, 'Net::HTTPTooManyRequests')
     
       expected SolanaRpcRuby::ApiError with "Net::HTTPTooManyRequests", got #<SolanaRpcRuby::ApiError: exception class/object expected> with backtrace:
         # ./lib/solana_rpc_ruby/api_client.rb:64:in `rescue in call_api'
         # ./lib/solana_rpc_ruby/api_client.rb:35:in `call_api'
         # ./spec/lib/solana_rpc_ruby/api_client_spec.rb:98:in `block (6 levels) in <top (required)>'
         # ./spec/lib/solana_rpc_ruby/api_client_spec.rb:97:in `block (5 levels) in <top (required)>'
     # ./spec/lib/solana_rpc_ruby/api_client_spec.rb:97:in `block (5 levels) in <top (required)>'
```